frontend/src/static/css/_extra.css

```
@@ -87,7 +87,7 @@ body.dark_theme {
  */
 
 html body {
-	font-family: 'Amulya', Arial, sans-serif;
+	font-family: 'Inter', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)